test/features/caseView_printButton.feature

```diff
@@ -0,0 +1,23 @@
+@functional @caseView
+Feature: Set of scenarios testing case view functionality
```

Comment:
.. case view print button functionality...?

Reply:
not mine, but will update